src/pretalx/cfp/forms/submissions.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, *args, **kwargs):
 
         super().__init__(*args, **kwargs)
 
-        queryset = self.event.questions.all()
+        queryset = self.event.questions.filter(active=True)
```

Comment:
Is it maybe possible to [use `QuestionManager.use_for_related_fields`](https://stackoverflow.com/a/7490852/2486196), so related managers also use `QuestionManager` (active questions only) instead of the default one returning all questions?

Reply:
Ah, no, the change in this line is actually unnecessary, as I (later) made the `active_question` manager the default manager, as visible [here](https://github.com/pretalx/pretalx/blob/8c1f4716f392181d6c9df16b55ba19a56a360eea/src/pretalx/submission/models/question.py#L92)

Reply:
https://github.com/pretalx/pretalx/commit/2a426aa488f8d311b91307cb79800c893203edc3 🎉